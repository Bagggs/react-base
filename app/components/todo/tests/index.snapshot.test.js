import React from 'react';
import renderer from 'react-test-renderer';
import Todo from '../';

describe('Todo', () => {
  it('renders correctly', () => {
    const todo = { id: 1, isComplete: false, name: 'Something to do' };
    const todoComponent = renderer.create(<Todo todo={ todo } />);

    expect(todoComponent.toJSON()).toMatchSnapshot();
  });
});
