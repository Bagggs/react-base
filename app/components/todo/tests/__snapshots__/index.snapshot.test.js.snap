// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo renders correctly 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <Row
    bsClass="row"
    className="show-grid"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
      md={8}
    >
      <h2>
        Todo List
      </h2>
    </Col>
    <Col
      bsClass="col"
      componentClass="div"
      md={4}
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="primary"
        className="btn btn-primary pull-right spacingTop"
        disabled={false}
        onClick={[Function]}
      >
        New Task
      </Button>
    </Col>
  </Row>
  <Row
    bsClass="row"
    className="show-grid"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
      md={6}
    >
      <h3
        className="spacing-bottom"
      >
        Incomplete
      </h3>
      <TodoList
        deleteTodo={[Function]}
        isComplete={false}
        openModal={[Function]}
        todos={Array []}
        updateTodo={[Function]}
      />
    </Col>
    <Col
      bsClass="col"
      componentClass="div"
      md={6}
    >
      <h3
        className="spacing-bottom"
      >
        Complete
      </h3>
      <TodoList
        deleteTodo={[Function]}
        isComplete={true}
        openModal={[Function]}
        todos={Array []}
        updateTodo={[Function]}
      />
    </Col>
  </Row>
</Grid>
`;
